src/components/TodosLogic.jsx

```diff
@@ -4,6 +4,7 @@ import InputTodo from '@/components/InputTodo';
 import TodosList from '@/components/TodosList';
 
 const TodosLogic = () => {
+  // eslint-disable-next-line no-use-before-define
```

Comment:
It actually work well but please let work without disabling our linters for good code practices